cnames_active.js

```diff
@@ -520,6 +520,7 @@ var cnames_active = {
   "chronosis": "jack-weilage.github.io/chronosis",
   "citation": "citation-js.github.io/site",
   "citeapa": "natsuozawa.github.io/citeapa",
+  "circle-ide": "circle-ide.github.io/editor.html"
```

Comment:
```suggestion
  "circle-ide": "circle-ide.github.io",
```
You cannot target a specific file